src/main/java/jalau/at18/kingoftokyo/DiceFaceKeeperProcess.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.HashMap;
+
+public class DiceFaceKeeperProcess {
+    private DiceFaceKeeper diceFaceKeeper;
+    private HashMap<DiceFace, Integer> diceFaceCount;
+
+    public DiceFaceKeeperProcess(DiceFaceKeeper diceFaceKeeper) {
```

Comment:
the same as DiceFaceKeeperProcess, it could be DicesResult....and the name could be smaller.